ui/static/content/objects/mul.md

```
@@ -0,0 +1,22 @@
+Multiply incoming values by a number.
+
+## Usage
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)